### ISSUE TITLE
destroy() will keep slides shown on page

### ISSUE THREAD

**Issue by MartijnHarmenzon** [NickPiscitelli/Glider.js#227]
The destroy() function will leave the slides shown on the page.
What is the best way to remove all images?
What is the functionality of the destroy() function?

**Comment by NickPiscitelli** [comment]
Yeah the idea behind destroy is to remove the slider and convert to a standard image gallery.Removing all slides can be accomplished by setting the containers HTML to an empty string and calling refresh()---- On Mon, 14 Mar 2022 11:23:50 -0400  Martijn ***@***.***> wrote ---- 
The destroy() function will leave the images shown on the page. What is the best way to remove all images?
